spec/requests/webhooks_spec.rb

```diff
@@ -158,21 +132,9 @@ def get_donation_detail(seller_id:)
           let(:item_type) { 'donation' }
 
           it 'creates pool donation and distrubutes the cents' do
-            donation_to_seller1 = get_donation_detail(seller_id: seller1.id)
-            donation_to_seller2 = get_donation_detail(seller_id: seller2.id)
-            donation_to_seller3 = get_donation_detail(seller_id: seller3.id)
-
-            expect(DonationDetail.count).to eq(3)
-            expect(Item.count).to eq(3)
-            expect(PaymentIntent.count).to eq(1)
-
-            expect(donation_to_seller1).not_to be_nil
-            expect(donation_to_seller2).not_to be_nil
-            expect(donation_to_seller2).not_to be_nil
-
-            expect(donation_to_seller1.amount).to eq(112)
-            expect(donation_to_seller2.amount).to eq(112)
-            expect(donation_to_seller3.amount).to eq(111)
+            expect(seller1.donation_amount).to eq(1_12)
+            expect(seller3.donation_amount).to eq(1_12)
+            expect(seller2.donation_amount).to eq(11_11)
```

Comment:
If we didn't have the sort on amount_raised, then it would have given the extra cent to sellers 1 and 2 because I think since the default sort is by id